acceptance/tests/environment/use_environment_from_environmentpath.rb

```diff
@@ -1,33 +1,34 @@
-test_name "Use environments from the environmentpath"
-require 'puppet/acceptance/classifier_utils'
-extend Puppet::Acceptance::ClassifierUtils
+test_name 'Use environments from the environmentpath' do
+  require 'puppet/acceptance/classifier_utils'
+  extend Puppet::Acceptance::ClassifierUtils
 
-tag 'audit:medium',
-    'audit:integration',
-    'server'
+  tag 'audit:medium',
+      'audit:integration',
+      'server'
 
-classify_nodes_as_agent_specified_if_classifer_present
+  classify_nodes_as_agent_specified_if_classifer_present
 
-testdir = create_tmpdir_for_user master, 'use_environmentpath'
+  testdir = create_tmpdir_for_user(master, 'use_environmentpath')
 
-def generate_environment(path_to_env, environment)
-  env_content = <<-EOS
+  def generate_environment(path_to_env, environment)
+    env_content = <<-EOS
   "#{path_to_env}/#{environment}":;
   "#{path_to_env}/#{environment}/manifests":;
   "#{path_to_env}/#{environment}/modules":;
-  EOS
-end
+    EOS
+    env_content
```

Comment:
Yep, Rubymine was busy complaining that we had assigned a variable, but hadn't used it. I picked adding the variable for the implied return, rather than removing the variable assignment.